### PR TITLE
Change representation of class file to match JVM

### PR DESCRIPTION
**Pull request by 1001mei**

### CODE REVIEW

File: src/compiler/__tests__/__utils__/test-utils.ts [hunk 1]
```diff
@@ -44,6 +44,6 @@ export function runTest(program: string, expectedLines: string[]) {
   expect(actualLines).toStrictEqual(expectedLines);
 }
 
-describe("compiler's test utils", () => {
-  it("Prevent jest from complaining", () => { });
-});
\ No newline at end of file
+//describe("compiler's test utils", () => {
```
Comment (Chang-CH) [inline src/compiler/__tests__/__utils__/test-utils.ts:47]: Maybe remove this?

**Review by Chang-CH — CHANGES_REQUESTED** [review]
Minor nitpicks, looks good otherwise

File: src/compiler/__tests__/__utils__/test-utils.ts (hunk 1 above)
Comment (1001mei) [reply src/compiler/__tests__/__utils__/test-utils.ts]: Removed.

**Review by Chang-CH — APPROVED** [review]
LGTM

**Pull request merged**